custom/icds/tasks/sms.py

```diff
@@ -51,3 +55,27 @@ def send_monthly_sms_report():
         send_html_email_async.delay(subject, recipients, message,
                                     email_from=settings.DEFAULT_FROM_EMAIL)
         raise e
+
+
+@receiver(post_save, sender=DailyOutboundSMSLimitReached)
+def send_sms_limit_exceeded_alert(sender, instance, **kwargs):
+    if settings.SERVER_ENVIRONMENT == 'icds':
```

Comment:
You should use `settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS`

Reply:
I also found some examples of `settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS` in HQ

But on a previous comment you asked, 
>Do we need to cover all domains on icds environment?

which is a valid point, we only need it in ICDS as of now. Do you still think we should go with `settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS`

Reply:
We can use `custom.icds.view_utils.is_icds_cas_project` now which would address both changes.

Reply:
Done